cmd/init.go

```diff
@@ -327,6 +327,7 @@ func init() {
 	initCmd.Flags().StringVar(&initOptions.ContractAddress, "contract-address", "", "Do not automatically deploy a contract, instead use a pre-configured address")
 	initCmd.Flags().StringVar(&initOptions.RemoteNodeURL, "remote-node-url", "", "For cases where the node is pre-existing and running remotely")
 	initCmd.Flags().Int64Var(&initOptions.ChainID, "chain-id", 2021, "The chain ID (Ethereum only) - also used as the network ID")
+	initCmd.Flags().StringVar(&initOptions.Network, "network", "mainnet", "The network to connect to (Cardano only)")
```

Comment:
Just FYI for when we get this into the main hyperledger repo would rather use `remote-node-url`

Reply:
@EnriqueL8 this is not a URL, it is the name of a network.